XouDevSpec/Classes/Model/ApiCallerModel.swift

```diff
@@ -10,6 +10,7 @@ import Foundation
 public enum AnimeError: Error {
     case noDataAvailable
     case cannotProcessData
+    case ProblemWithUrl
```

Comment:
"problemWithUrl" -> for naming convention and consistency

Reply:
done and done